jsk_mycobot_robot/README.md

```diff
@@ -0,0 +1,102 @@
+# jsk_mycobot_robot
+
+ROS package for Mycobot ARM robot.
+
+- Mycobot web page
+  https://www.elephantrobotics.com/en/mycobot-en/
+- GitHub
+  https://github.com/elephantrobotics/mycobot_ros
+
+## How to setup the Mycobot robot
+
+under construction 
+
+## How to setup development environment
+
+### My Setup
+
+We need `pymycobot` for conection with M5Stack for the arm. This modules is supported in both python2 and python3
+
+```bash
+pip install pymycobot
+```
+
+### ROS Environment Setup
+
+Use `wstool`, `rosdep` and `catkin` to checkout and compile the source tree.
+
+```bash
+mkdir -p ~/mycobot_ws/src
+cd ~/mycobot_ws/src
+wstool init
+wstool merge https://raw.githubusercontent.com/tongtybj/jsk_robot/PR/mycobot/eus/jsk_mycobot_robot/mycobot.rosinstall
```

Comment:
@tongtybj I mean please replace this line with https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_mycobot_robot/mycobot.rosinstall, because we do not want to create another PR after we merge this PR, just to update .rosinstall URL

Reply:
Should I change this line right now? 
I thought I should do this just before this PR is going to be merged.

Reply:
yes please update in this PR

Reply:
Updated